tests/unit/resources/sample.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    service: <name>
+spec:
+  containers:
+    - name: <name>
```

Comment:
Why do we have angular brackets here? I.e., why `<name>` ?

Reply:
Same as above

Reply:
Ok